internal/install/command.go

```diff
@@ -12,8 +12,6 @@ import (
 	"github.com/newrelic/newrelic-client-go/newrelic"
 )
 
-var configFiles []string
```

Comment:
Let's pull this out and eventually replace it with a recipe search parameter to be forwarded to the service.